packages/fxa-auth-server/lib/account-delete.ts

```diff
@@ -11,11 +13,18 @@ import { PayPalHelper } from './payments/paypal/helper';
 import { StripeHelper } from './payments/stripe';
 import push from './push';
 import pushboxApi from './pushbox';
-import { AuthLogger } from './types';
+import { AppConfig, AuthLogger } from './types';
+/*
+import {
+  uid as uidValidator,
+  customerId as customerIdValidator,
+} from './routes/validators';
+//*/
```

Comment:
The issue's acceptance criteria asked for task payload validation but since this patch is for creating tasks I did not end up using the code.  I've commented out a couple blocks so that the linter won't complain about unused code.

Reply:
Ah, yea, that should be handled when getting the task, not sending it.